hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -536,24 +560,62 @@ public boolean scheduleExists(ScheduleID id) {
 
 		List<TokenNftInfo> nftInfos = new ArrayList<>();
 		var uniqueTokensMap = uniqueTokens.get();
-		uniqueTokenAccountOwnerships.get()
-				.get(fromGrpcAccountId(aid), (int) start, (int) end).forEachRemaining(nftId -> {
-			var nft = uniqueTokensMap.get(nftId);
-			nftInfos.add(
-					TokenNftInfo.newBuilder()
-							.setAccountID(aid)
-							.setCreationTime(nft.getCreationTime().toGrpc())
-							.setNftID(NftID.newBuilder()
-									.setTokenID(nftId.tokenId().toGrpcTokenId())
-									.setSerialNumber(nftId.serialNumber())
-									.build())
-							.setMetadata(ByteString.copyFrom(nft.getMetadata()))
-							.build()
-			);
-		});
+		var uniqueTokenAccountCount = uniqueTokenAccountOwnerships.get().getCount(fromGrpcAccountId(aid));
```

Comment:
I think that it would be better if have another method `computeAssociationRanges` that would return the `start/end` indexes for both the 2 FCOTMR

That way the code would be cleaner imo. If there is no need to query a map we could return `0, 0` for start/end. 

If we have that the logic afterwards will be very clean